mscore/startcenter.ui

```diff
@@ -106,8 +106,6 @@
    <container>1</container>
   </customwidget>
  </customwidgets>
- <resources>
-  <include location="musescore.qrc"/>
- </resources>
+ <resources/>
```

Comment:
Was this intentional? It seems unrelated to the purpose of the PR.

Reply:
A side effect that I left in deliberatly.